.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    -   id: no-commit-to-branch
+    -   id: check-merge-conflict
+    -   id: check-toml
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
+-   repo: https://github.com/pycqa/isort
```

Comment:
I think we should discuss `isort` and its implications in a separate PR... Furthermore, I don't think we should add it as a pre-commit hook before it is part of our workflow checks.

Reply:
Understandable and agree